### PR TITLE
concept string methods minor arrangement

### PR DESCRIPTION
**Pull request by valentin-p**
* Delete the str-method concept which is now string-method
* Improve introduction in exercise and concept
* Add couple unit tests
* Rename the unit test class to PascalCase

### CODE REVIEW

**Review by BethanyG — CHANGES_REQUESTED** [review]
Thank you for this, @valentin-p!  🎉  🌈   Some of these are suggestions, and some are a bit stronger.  Ping me if you have questions or issues.

File: exercises/concept/little-sisters-essay/.docs/instructions.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
-In this exercise you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent word choice.
+In this exercise, you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent choice of words.
```
[the remark this answers is not in the document]
Comment (valentin-p) [reply exercises/concept/little-sisters-essay/.docs/instructions.md:1]: Wow, true, but weird. Thanks

**Comment by valentin-p** [comment]
Part of #2286

File: exercises/concept/little-sisters-essay/.docs/instructions.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-In this exercise you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent word choice.
+In this exercise you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent choice of words.
```
Comment (BethanyG) [inline exercises/concept/little-sisters-essay/.docs/instructions.md:1]:
```suggestion
In this exercise you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent word choice.
```

File: exercises/concept/little-sisters-essay/.docs/instructions.md [hunk 3]
```diff
@@ -22,7 +22,8 @@ True
 
 ## 3. Clean up spacing
 
-To make the paper look professional, unnecessary spacing needs to be removed. Implement the function `remove_extra_spaces()` which takes in the parameter `sentence` and removes extra whitespace at the beginning and at the end of the sentence. This function should return a `str`.
+To make the paper _even better_, you can replace some of the adjectives with synonyms. Write the function `replace_word_choice()` that takes `sentence`, `old_word`, and `new_word` as parameters.
+This function should replace all instances of the `old_word` with the `new_word`, and return a new `str` with the updated sentence.
```
Comment (BethanyG) [inline exercises/concept/little-sisters-essay/.docs/instructions.md:26]:
```suggestion
```

File: exercises/concept/little-sisters-essay/.docs/instructions.md [hunk 4]
```diff
@@ -22,7 +22,8 @@ True
 
 ## 3. Clean up spacing
 
-To make the paper look professional, unnecessary spacing needs to be removed. Implement the function `remove_extra_spaces()` which takes in the parameter `sentence` and removes extra whitespace at the beginning and at the end of the sentence. This function should return a `str`.
+To make the paper _even better_, you can replace some of the adjectives with synonyms. Write the function `replace_word_choice()` that takes `sentence`, `old_word`, and `new_word` as parameters.
```
Comment (BethanyG) [inline exercises/concept/little-sisters-essay/.docs/instructions.md:25]:
```suggestion
To make the paper look professional, unnecessary spacing needs to be removed. Implement the function `remove_extra_spaces()` which `sentence` as a parameter, and removes extra whitespace at the beginning and at the end of the sentence. This function should return a `str`.
```

File: exercises/concept/little-sisters-essay/.docs/instructions.md (hunk 3 above)
Comment (BethanyG) [reply exercises/concept/little-sisters-essay/.docs/instructions.md:27]: I think this was a mis-cut and paste.

File: exercises/concept/little-sisters-essay/.docs/instructions.md [hunk 5]
```diff
@@ -22,7 +22,9 @@ True
 
 ## 3. Clean up spacing
 
-To make the paper look professional, unnecessary spacing needs to be removed. Implement the function `remove_extra_spaces()` which takes in the parameter `sentence` and removes extra whitespace at the beginning and at the end of the sentence. This function should return a `str`.
+To make the paper look professional, unnecessary spacing needs to be removed. Implement the function `remove_extra_spaces()` which `sentence` as a parameter, and removes extra whitespace at the beginning and at the end of the sentence. 
+This function should return a `str`.
+This function should replace all instances of the `old_word` with the `new_word`, and return a new `str` with the updated sentence.
```
Comment (BethanyG) [inline exercises/concept/little-sisters-essay/.docs/instructions.md:27]: This sentence needs to be removed....it belongs to the task below.

Comment (BethanyG) [inline exercises/concept/little-sisters-essay/.docs/instructions.md:27]:
```suggestion
```

Comment (BethanyG) [reply exercises/concept/little-sisters-essay/.docs/instructions.md:27]: This sentence needs to be removed....it belongs to the task below.

**Review by BethanyG — APPROVED** [review]

**Pull request merged**